src/proxy-page/proxy-page.service.ts

```diff
@@ -113,12 +117,8 @@ export class ProxyPageService {
     }
     delUnnecessaryCode()(this.context);
     addCustomCss(this.config, style)(this.context);
-    addMessageBus(this.config)(this.context);
-    if (nozoom == undefined && view !== 'iframe-resizer') {
-      addZooming(this.config)(this.context);
-      addNoZoom()(this.context);
-    }
-    addHighlightjs(this.config)(this.context);
+    addLibrariesCSS()(this.context);
```

Comment:
The implementation looks good, but do you have some ideas for an automated test strategy for this new function? (also for `addLibrariesJS()`). Can any of the existing tests be used as a model?

Reply:
I had the same thought but not with an obvious outcome.
On one hand we could just check that the libraries are properly added in the HTML head and body structure. That is the obvious and simple unit test.
Going beyond may imply some initialization of the libraries or checking some basic and mandatory styles, but this could be quite oriented and to end up quite complex.